src/components/List.tsx

```diff
@@ -0,0 +1,66 @@
+import classNames from "classnames";
+import { nullValuesAsUndefined } from "null-as-undefined";
+import PropTypes from "prop-types";
+import React from "react";
+
+import "lbh-frontend/lbh/core/_lists.scss";
+
+/**
+ * The types of {@link List}.
+ */
+export enum ListTypes {
+  Bullet = "bullet",
+  Number = "number"
+}
+
+/**
+ * The proptypes for {@link List}.
+ */
+export interface ListProps {
+  id?: string | null;
+  className?: string | null;
+  /**
+   * Defaults to an unordered list if no type is specified.
+   */
+  type?: ListTypes | null;
+  /**
+   * The items in the list. Can be anything from a string to another list
+   * component.
+   */
+  items: React.ReactNode[];
+}
+
+/**
+ * A list of items.
+ */
+export const List: React.FunctionComponent<ListProps> = (
+  props: ListProps
+): JSX.Element => {
+  const { id, className, type, items } = nullValuesAsUndefined(props);
+
+  const ListType = type == ListTypes.Number ? "ol" : "ul";
+
+  return (
+    <ListType
+      id={id}
+      className={classNames(
+        "govuk-list",
+        "lbh-list",
+        { "lbh-list--number govuk-list--number": type == ListTypes.Number },
```

Comment:
```suggestion
        { "lbh-list--number govuk-list--number": type === ListTypes.Number },
```

Better to be in this habit.